gitops/rings/README.md

```diff
@@ -0,0 +1,144 @@
+# Rings
```

Comment:
Do we have any high level docs explaining the concepts of rings we inject in the intro?